idlpy/src/types.c

```diff
@@ -699,8 +848,19 @@ emit_const(
 
     const idl_literal_t *literal = ((const idl_const_t *)node)->const_expr;
 
+    //printf("------- emit_const: name %s \n",idl_identifier(node));
```

Comment:
Please don't leave commented code in the PR.

Reply:
After reviewing it today, all the printfs we have added have been deleted.